staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -35,6 +35,9 @@ type DelegatingAuthorizationOptions struct {
 	// SubjectAccessReview.authorization.k8s.io endpoint for checking tokens.
 	RemoteKubeConfigFile string
 
+	// DisableAuthorization is used to explicitly disable any authorization, mostly for development purposes.
+	DisableAuthorization bool
```

Comment:
Why would I want this lever?  What's hard about doing development with a pod and image?

Reply:
we want to extract openapi from a server, i.e. just launch the server, do one curl and terminate.

Reply:
and about pod and image: people want to run an apiserver inside an idea. a multi-process setup is far too complicated. Am happy to call this option explicitly "dev-only" or something like that.

Reply:
> Am happy to call this option explicitly "dev-only" or something like that.

Thing is, I really don't even want devs using it.  Lack of consideration for security has produced sins we're **still** paying for after 3 years.  What if you made it `system:masters` only and used a client cert.  That gives you the single file.

Reply:
If it starts up without a config, but only accepts `system:masters`, that sounds fine, works for development, but any non-dev use-case immediately need a secure setup. Is this was what you mean? If not, please elaborate.